mainnet/config.yaml

```diff
@@ -9,155 +15,124 @@ TERMINAL_TOTAL_DIFFICULTY: 8626000000000000000000058750000000000000000000
-DOMAIN_BEACON_PROPOSER: 0x00000000
-DOMAIN_BEACON_ATTESTER: 0x01000000
-DOMAIN_RANDAO: 0x02000000
-DOMAIN_DEPOSIT: 0x03000000
-DOMAIN_VOLUNTARY_EXIT: 0x04000000
-DOMAIN_SELECTION_PROOF: 0x05000000
-DOMAIN_AGGREGATE_AND_PROOF: 0x06000000
-DOMAIN_SYNC_COMMITTEE: 0x07000000
-DOMAIN_SYNC_COMMITTEE_SELECTION_PROOF: 0x08000000
-DOMAIN_CONTRIBUTION_AND_PROOF: 0x09000000
+# Some forks are disabled for now:
+#  - These may be re-assigned to another fork-version later
+#  - Temporarily set to max uint64 value: 2**64 - 1
 
 # Altair
 ALTAIR_FORK_VERSION: 0x01000064
 ALTAIR_FORK_EPOCH: 512
 # Bellatrix
 BELLATRIX_FORK_VERSION: 0x02000064
-BELLATRIX_FORK_EPOCH: 385536 # 2022-11-30T19:23:40.000Z
+BELLATRIX_FORK_EPOCH: 385536  # 2022-11-30T19:23:40.000Z
 # Capella
 CAPELLA_FORK_VERSION: 0x03000064
-CAPELLA_FORK_EPOCH: 648704 # 2023-08-01T11:34:20.000Z
+CAPELLA_FORK_EPOCH: 648704  # 2023-08-01T11:34:20.000Z
 # Deneb
 DENEB_FORK_VERSION: 0x04000064
 DENEB_FORK_EPOCH: 18446744073709551615
+# EIP6110
+EIP6110_FORK_VERSION: 0x05000064  # temporary stub
+EIP6110_FORK_EPOCH: 18446744073709551615
+# EIP7002
+EIP7002_FORK_VERSION: 0x05000064  # temporary stub
+EIP7002_FORK_EPOCH: 18446744073709551615
+# WHISK
+WHISK_FORK_VERSION: 0x06000064  # temporary stub
+WHISK_FORK_EPOCH: 18446744073709551615
+
+
+# Time parameters
+# ---------------------------------------------------------------
+# 5 seconds
+SECONDS_PER_SLOT: 5
+# 6 (estimate from xDai mainnet)
+SECONDS_PER_ETH1_BLOCK: 6
+# 2**8 (= 256) epochs ~27 hours
+MIN_VALIDATOR_WITHDRAWABILITY_DELAY: 256
+# 2**8 (= 256) epochs ~27 hours
+SHARD_COMMITTEE_PERIOD: 256
+# 2**10 (= 1024) ~1.4 hour
+ETH1_FOLLOW_DISTANCE: 1024
+
 
+# Validator cycle
+# ---------------------------------------------------------------
+# 2**2 (= 4)
 INACTIVITY_SCORE_BIAS: 4
 # 2**4 (= 16)
 INACTIVITY_SCORE_RECOVERY_RATE: 16
+# 2**4 * 10**9 (= 16,000,000,000) Gwei
+EJECTION_BALANCE: 16000000000
+# 2**2 (= 4)
+MIN_PER_EPOCH_CHURN_LIMIT: 4
+# 2**12 (= 4096)
+CHURN_LIMIT_QUOTIENT: 4096
+# [New in Deneb:EIP7514] 2*
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 2
+
+# Fork choice
+# ---------------------------------------------------------------
+# 40%
+PROPOSER_SCORE_BOOST: 40
+# 20%
+REORG_HEAD_WEIGHT_THRESHOLD: 20
+# 160%
+REORG_PARENT_WEIGHT_THRESHOLD: 160
+# `2` epochs
+REORG_MAX_EPOCHS_SINCE_FINALIZATION: 2
+
+
+# Deposit contract
+# ---------------------------------------------------------------
+# xDai Mainnet
+DEPOSIT_CHAIN_ID: 100
+DEPOSIT_NETWORK_ID: 100
+DEPOSIT_CONTRACT_ADDRESS: 0x0B98057eA310F4d31F2a452B414647007d1645d9
+
+# Networking
+# ---------------------------------------------------------------
+# `10 * 2**20` (= 10485760, 10 MiB)
+GOSSIP_MAX_SIZE: 10485760
+# `2**10` (= 1024)
+MAX_REQUEST_BLOCKS: 1024
+# `2**8` (= 256)
+EPOCHS_PER_SUBNET_SUBSCRIPTION: 256
+# `MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2` (= 33024, ~5 months)
+MIN_EPOCHS_FOR_BLOCK_REQUESTS: 33024
```

Comment:
This used to be 2304 before this PR. Intended change? The new number does not match the comment anymore.

Reply:
Actually, it was 33024 already before (inherited from the preset). But the comment still is wrong and suggests 2304.

Reply:
https://github.com/status-im/nimbus-eth2/pull/5590

Fixed it in Nimbus. Because `config.yaml` didn't specify it, we defaulted to 2304 based on the computation in the comment.

Reply:
The variable `MIN_EPOCHS_FOR_BLOCK_REQUESTS` was not defined in the config before this change. It was added 6 months ago in Ethereum's config on a effort to define network variables https://github.com/ethereum/consensus-specs/commit/f87e1436bfbc1f51a18aedc337c6013cd3b3305e

Thanks for noticing, does Nimbus prune blocks according to `MIN_EPOCHS_FOR_BLOCK_REQUESTS`?